examples/jvm/defaults/build.gradle.kts

```diff
@@ -0,0 +1,86 @@
+plugins {
+    kotlin("jvm") version "1.7.10"
```

Comment:
In the future, it may be worth making a generator of examples; depending on whether their number will increase.

At the moment, in order not to forget to change the versions of Kover, there is a special check (see kotlinx.kover.gradle.plugin.test.functional.examples.ExamplesTests#checkVersions)